content/kapacitor/v1.6/event_handlers/kafka.md

```diff
@@ -221,3 +271,29 @@ Add the handler:
 ```bash
 kapacitor define-topic-handler kafka_cpu_handler.yaml
 ```
+
+### Using SASL with Kapacitor
+
+Kapacitor can be configured to use SASL if you want to use methods of authentication other than SSL. 
+An example would be using Kapacitor to authenticate directly against Kafka with a username/password. 
```

Comment:
For example, to use Kapacitor to authenticate with a username/password directly against Kafka, configure SASL.